h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -203,13 +241,15 @@ class SearchBarController extends Controller {
         this.setState({open: false});
       }
       closeDropdown();
+      this._input.removeEventListener('keydown', setupListenerKeys,
```

Comment:
You're right. I did that on purpose because I thought that's what the code was doing. When I wrote that I thought setupListenerKeys was a function assigning other event listeners. My oversight. Perhaps renaming this to something like `handleKeys` would help make that more obvious